package.json

```diff
@@ -14,7 +14,9 @@
   },
   "dependencies": {
     "@nuxt/http": "^0.6.4",
```

Comment:
should remove the http libarary since we're using the axios one.

Reply:
yep, good call

Reply:
done